aibolit/__main__.py

```diff
@@ -330,54 +330,60 @@ def inference(
 
     return results
 
-
-def run_recommend_for_file(file: str, args):
+# flake8: noqa: C901
```

Comment:
can we not ignore it?

Reply:
after 1 if statement was added, the function began to show it. It is difficult to split it